daemon/container.go

```diff
@@ -54,6 +54,19 @@ func (daemon *Daemon) GetContainer(prefixOrName string) (*container.Container, e
 	return daemon.containers.Get(containerID), nil
 }
 
+func (daemon *Daemon) GetcontainerCondition(prefixOrName string, conditions ...func(*container.Container) error) (*container.Container, error) {
```

Comment:
The case may be changed, e.g., `s/GetcontainerCondition/GetContainerCondition/`, ?